python/pyspark/pandas/utils.py

```diff
@@ -464,20 +464,12 @@ def is_testing() -> bool:
     return "SPARK_TESTING" in os.environ
 
 
-def default_session(conf: Optional[Dict[str, Any]] = None) -> SparkSession:
-    if conf is None:
-        conf = dict()
+def default_session() -> SparkSession:
```

Comment:
I'm not sure we can remove the `conf` argument here?
I guess we should show a deprecation warning if it's not `None` for now and remove it in the future?

Reply:
I think this isn't an API that's not documented so it should be fine.

Reply:
I'd just leave it to you.